native/acter/api.rsh

```diff
@@ -592,6 +604,52 @@ object Rsvp {
     fn status() -> string;
 }
 
+//  ### ##   ### ###    ##      ## ##   #### ##    ####    ## ##   ###  ##   ## ##   
+//  ##  ##   ##  ##     ##    ##   ##  # ## ##     ##    ##   ##    ## ##  ##   ##  
+//  ##  ##   ##       ## ##   ##         ##        ##    ##   ##   # ## #  ####     
+//  ## ##    ## ##    ##  ##  ##         ##        ##    ##   ##   ## ##    #####   
+//  ## ##    ##       ## ###  ##         ##        ##    ##   ##   ##  ##      ###  
+//  ##  ##   ##  ##   ##  ##  ##   ##    ##        ##    ##   ##   ##  ##  ##   ##  
+// #### ##  ### ###  ###  ##   ## ##    ####      ####    ## ##   ###  ##   ## ## 
+
+
+object ReactionManager {
+
+    /// get my reaction status
+    fn my_status() -> Future<Result<bool>>;
```

Comment:
this is a bit vague. What does `my_status = true` mean? I assume that is meant to imply the user has reacted? then let's call it that `have_i_reacted`...